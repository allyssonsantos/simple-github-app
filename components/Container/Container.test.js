import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Container from './';

describe('With Snapshot Testing', () => {
  it('Should match snapshot', () => {
    const component = renderer.create(<Container>Hello</Container>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
