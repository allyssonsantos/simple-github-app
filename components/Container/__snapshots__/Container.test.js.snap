// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Should match snapshot 1`] = `
.c0 {
  margin: 0 auto;
  max-width: 700px;
}

<div
  className="c0"
>
  Hello
</div>
`;
